manifests/upstream/keptn/helm/generated_templates/keptn/charts/control-plane/charts/mongodb/templates/svc.yaml

```diff
@@ -0,0 +1,21 @@
+---
+# Source: keptn/charts/control-plane/charts/mongodb/templates/svc.yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: mongodb
+  labels:
+    app.kubernetes.io/name: mongodb
+    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/managed-by: Helm
```

Comment:
Can we strip these labels from all helm resources?

Reply:
I believe I have done that before so yes. Let me try.

Reply:
I think the best way to handle this is to use `commonLabels`
```
commonLabels:
  app.kubernetes.io/part-of: keptn
  app.kubernetes.io/managed-by: karina
  app.kubernetes.io/instance: keptn
```

I can write a patch to remove these labels completely but it is complicated and cumbersome (need to patch labelSelector + labels of pod templates for all deployment and statefulset too)

Reply:
Can we not pass all kustomize output through yq to drop them?

Reply:
Can you remind me what version of `yq` we are using? Is this the one: https://github.com/mikefarah/yq? Version 4?